.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.19-openshift-4.12
+  tag: rhel-8-release-golang-1.22-openshift-4.17
```

Comment:
can we change to `rhel-9-release` ?`

Reply:
RHEL9 migration is something we need to do, not only in this repository. Unfortunately I don't think I'll have enough time to do it for this release (before Christmas break). So, we need to file a bug for RHEL9 migration similar to [what we did for the externaldns](https://issues.redhat.com//browse/OCPBUGS-41683).

Reply:
OK. then other look good.
@ShudiLi could you please help file a bug.

Reply:
@lihongan Filed https://issues.redhat.com/browse/OCPBUGS-46442, thanks.